Makefile

```diff
@@ -0,0 +1,38 @@
+.DEFAULT_GOAL = help
+
+.PHONY: sh
+sh: ## Get a terminal with Bash.
+	docker-compose exec server bash
+
+.PHONY: logs
+logs: ## Tail the logs, locally.
+	docker-compose logs -f
+
+.PHONY: up
+up: ## Spin up a dev suite of containers.
+	docker-compose up -d
+
+.PHONY: generate
+generate: ## a one time use command needed to setup the db after the creation of the containers
+	docker-compose exec -ti server npx prisma generate --schema server/prisma/schema.prisma 
+
+.PHONY: migrate
+migrate: ## push db changes to the db.
+	docker-compose exec -ti server npx prisma db push --schema server/prisma/schema.prisma
+
+.PHONY: down
+down: ## Stops up containers.
+	docker-compose down
+
+.PHONY: db
+db: ## attach to the db cli
+	docker-compose exec -ti mongo mongo --host=mongo --port=27018 -u "mongoadmin" -p "secret"
+
+.PHONY: create
+create: up generate migrate ## create a local env from nothing
+
+
+# https://www.freecodecamp.org/news/self-documenting-makefile/
+.PHONY: help
+help: ## Show this.
+	@egrep -h '\s##\s' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-24s\033[0m %s\n", $$1, $$2}'
```

Comment:
I added it to the docs as well, but this is a handy way to run sets of command line commands


for example, if you have docker installed, you can run `make create` and you'll get a working version of the app up and running with a local db for testing